Source/JavaScriptCore/assembler/ProbeStack.h

```diff
@@ -139,7 +139,9 @@ class Page {
     static_assert(s_pageSize > s_chunkSize, "bad pageSize or chunkSize");
     static_assert(s_chunkSize == (1 << s_chunkSizeShift), "bad chunkSizeShift");
 
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
```

Comment:
Do you mind opening a bug to track migration from `std::aligned_storage`?

Reply:
It looks like there's very valid reason to drop this.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=274966